solr-search/src/main/java/edu/unc/lib/dl/search/solr/model/HierarchicalBrowseResultResponse.java

```diff
@@ -67,7 +68,7 @@ public void setSubcontainerCounts(Map<String, Long> subcontainerCounts) {
     }
 
     public void populateSubcontainerCounts(List<FacetField> facetFields) {
-        subcontainerCounts = new HashMap<String, Long>();
+        subcontainerCounts = new HashMap<>();
```

Comment:
since subcontainerCounts is already initialized in the constructor, do you also want it set to a fresh hashMap here?